practice-two/src/index.html

```diff
@@ -315,14 +315,73 @@ <h3 class="project__item-title">Watch apps</h3>
         </div>
       </section>
       <!-- END TPROJECTS SECTION-->
-
+ 
       <!-- START REVIEW SECTION -->
       <section></section>
       <!-- END REVIEW SECTION-->
 
-      <!-- START PRCING SECTION -->
-      <section></section>
-      <!-- END PRCING SECTION-->
+     <!-- START PRCING SECTION -->
+    <section class="pricing">
+      <div class="container pricing__wrapper">
+        <h2 class="heading-section text-color-secondary pricing__title ">
+          Subscribing Plans
+        </h2>
+        <div class="pricing__item">
+          <div class="pricing__card">
+            <div class="pricing__card--freemium pricing__card-heading">
+              <p class="pricing__card-title">FREEMIUM</p>
+              <p class="pricing__card-price"><sup class="price-symbol">$</sup>0</p>
+            </div>
+              <ul class="pricing__card-description">
+                <li class="text-normal">Basic Collections</li>
+                <li class="text-normal">10 Messages</li>
+                <li class="text-normal">50 Images per Day</li>
+              </ul>
+            <button class="btn btn-base btn--pricing">GET STARTED</button>
+          </div>
+          <div  class="pricing__card">
+            <div class="pricing__card--start-up pricing__card-heading">
```

Comment:
You can add class text-center here instead of style center for pricing__card-title and pricing__card-price